Benchmark-Models/Laske_PLOSComputBiol2019/model_Laske_PLOSComputBiol2019.xml

```diff
@@ -0,0 +1,5172 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Created by COPASI version 4.30 (Build 240) on 2021-07-16 10:12 with libSBML version 5.19.0. -->
+<sbml xmlns="http://www.sbml.org/sbml/level2/version3" level="2" version="3">
+  <model metaid="COPASI0" id="Laske_et_al___Influenza_A_Modelling" name="Laske et al., Influenza A Modelling">
```

Comment:
i think this would be one of the last things to change
could you please adapt the `id` and `name` fields to have the same name as the benchmark model, i.e. Laske_PLOSComputBiol2019

Reply:
@C-Peiter 😄 ?

Reply:
Yeah, sorry. Was planning on doing that today or on the weekend so I forgot throughout the week. :smile:

Reply:
no worries 😉